Competition2017/src/competition/subsystems/shooter_wheel/ShooterWheelSubsystem.java

```diff
@@ -43,6 +44,9 @@ public ShooterWheelSubsystem(
                 propManager.createPersistentProperty(side + " flush to boiler target speed", 9000);
         trimFlushToBoilerSpeed =
                 propManager.createEphemeralProperty(side + " trim speed", 0.0);
+        optimalWheelSpeed = 
+                propManager.createPersistentProperty("Minimum wheel speed", 9001);
```

Comment:
Issa, I recommend here that you have a minimum wheel speed equal to some fraction of target speed (like, 75% of target speed) so it will work for all targets.

That way, when Andrew is running tests at speed level 2000, it will still work, and when we go back up to normal speeds (8000-10000) it will continue working.